support-frontend/assets/components/subscriptionCheckouts/stage.jsx

```diff
@@ -24,6 +22,16 @@ type PropTypes = {|
   countryGroupId: CountryGroupId,
 |};
 
+type StagePropTypes = {
+  stage: Stage,
+  formSubmitted: boolean,
+  countryGroupId: CountryGroupId,
```

Comment:
I don't think countryGroupId is used in checkStage

Reply:
Nice pick up! Removed 🥇